docs/references/vod/import.mdx

```diff
@@ -1,26 +1,79 @@
 ---
-title: Import an Asset
-description: Livepeer Studio API Reference | Import
-metaTitle: Import - Livepeer Studio
-metaDescription: Livepeer Studio API Reference | Import
+title: Upload with URL
 ---
 
-# Import an Asset
 
-Import a video `Asset` to Livepeer from an external URL through the
+Upload a video `Asset` to Livepeer Studio from an external URL through the
 `POST /api/asset/import` API.
 
-```jsx
+
+## Request
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+
+
+<Tabs>
+  <TabItem value="bash" label="bash">
+
+
+```bash
 curl --location --request POST 'https://livepeer.studio/api/asset/import' \
 --header 'Authorization: Bearer $API_TOKEN' \
 --header 'Content-Type: application/json' \
 --data-raw '{
     "url":"$EXTERNAL_URL",
     "name":"Example name"
 }'
+
 ```
 
-### Request body parameters
+
+</TabItem>
+<TabItem value="js" label="NodeJS">
+
+```js
+app.post("https://livepeer.studio/api/asset/request-upload", {
```

Comment:
Did you mean `axios` here? I'm not sure what is `app.post` in NodeJS.

Furthermore, I don't think we should have separate examples for "node" and "fetch". Can provide just the `fetch` example and devs can use [node-fetch](https://www.npmjs.com/package/node-fetch) if they want to make zero changes to the example. Or can have all examples with `axios` which can be used from browser or node (the downside is forcing it into every developer). It's just an example anyway, not necessarily something that should be ccopy-pasted and work out of the box on every environment, so I'd rather stick to `fetch` wherever possible which is the most standard client IMO.